docs/databases/swift-kuery-orm.html

```diff
@@ -170,7 +170,8 @@ <h4 class="heading-4">macOS</h4>
         <pre><code class="language-swift">brew install postgresql</code></pre>
         <h4 class="heading-4">Linux</h4>
         <p>On Linux we can use <strong>apt-get</strong> to install Postgres:</p>
-        <pre><code class="language-swift">sudo apt-get install libpq-dev</code></pre>
+        <pre><code class="language-swift">sudo apt install postgresql postgresql-contrib</code></pre>
```

Comment:
On the line above we say we can  use `apt-get` to install Postgres then in the new addition we say use `apt`?